### PR TITLE
Revise Swift SDK docs

### PR DESCRIPTION
**Pull request by bisgardo**
Try and actually follow the structure of the existing docs

### CODE REVIEW

File: source/mainnet/net/guides/wallet-sdk/wallet-sdk-credential-deployment.rst [hunk 1]
```diff
@@ -338,7 +302,18 @@ is the signing key that corresponds to the public key used when creating the tra
 
         Swift (iOS)
 
-        The Swift SDK for iOS is still in development.
+        .. code-block:: Kotlin
```
Comment (soerenbf) [inline source/mainnet/net/guides/wallet-sdk/wallet-sdk-credential-deployment.rst:305]:
```suggestion
        .. code-block:: Swift
```

I assume?

File: source/mainnet/net/guides/wallet-sdk/wallet-sdk-credential-deployment.rst [hunk 2]
```diff
@@ -413,6 +388,18 @@ If successful, the credential will be deployed, and it is now possible to start
 
     .. tab::
 
-        Swift (iOS)
+        Swift (macOS, iOS)
+
+        .. code-block:: Kotlin
```
Comment (soerenbf) [inline source/mainnet/net/guides/wallet-sdk/wallet-sdk-credential-deployment.rst:393]:
```suggestion
        .. code-block:: Swift
```

**Review by soerenbf — APPROVED** [review]
LGTM

File: source/mainnet/net/guides/wallet-sdk/wallet-sdk-account-transaction.rst [hunk 3]
```diff
@@ -237,4 +227,12 @@ Finally, when the transaction has been constructed and signed, it is ready to be
 
         Swift (iOS)
```
Comment (bisgardo) [inline source/mainnet/net/guides/wallet-sdk/wallet-sdk-account-transaction.rst:228]:
```suggestion
        Swift (macOS, iOS)
```

**Pull request merged**